.travis.yml

```
@@ -6,14 +6,27 @@ services:
 addons:
   postgresql: "10"
 apt:
+  update: true
   packages:
   - postgresql-10
   - postgresql-client-10
+  - python
+  - python-pip
+  - python-setuptools
+  - python-wheel
 dist: xenial
 cache:
   - bundler
```

```suggestion
   - bundler
   - pip
```
Its probably worth adding a pip to the caching options
https://docs.travis-ci.com/user/caching/#pip-cache